### PR TITLE
vsphere ipi deployment - not all nodes created

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu [vavuthu@redhat.com](mailto:vavuthu@redhat.com)

### CODE REVIEW

**Comment by vavuthu** [comment]
Verified
```
2024-07-16 20:01:34  10:31:34 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig get Node  -o yaml
2024-07-16 20:05:10  10:35:01 - MainThread - ocs_ci.utility.vsphere - INFO  - Powering on VMs: ['jnk-pr10116b7191-2hknt-worker-0-sbq5b']
2024-07-16 20:05:10  10:35:10 - MainThread - ocs_ci.utility.vsphere - INFO  - Waiting for VMs ['jnk-pr10116b7191-2hknt-worker-0-sbq5b'] to power on based on network connectivity. Current VMs IPs: [None]
.
.
.
2024-07-16 20:09:13  10:39:02 - MainThread - ocs_ci.ocs.node - WARNING  - waiting for 6 nodes to create but found 5 nodes
2024-07-16 20:09:13  10:39:02 - MainThread - ocs_ci.utility.utils - INFO  - Going to sleep for 60 seconds before next iteration
2024-07-16 20:10:09  10:40:02 - MainThread - ocs_ci.utility.utils - INFO  - Executing command: oc --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig get Node  -o yaml
2024-07-16 20:10:09  10:40:02 - MainThread - ocs_ci.ocs.node - INFO  - All 6 nodes are created successfully.

```

File: ocs_ci/ocs/node.py [hunk 1]
```diff
@@ -1754,6 +1761,29 @@ def verify_all_nodes_created():
         )
 
 
+def power_on_vms():
```
Comment (petr-balogh) [inline ocs_ci/ocs/node.py:1764]: What about rename to something like power_on_ocp_node_vms()

And mention to docstring that it makes sure all cluster node VMs are powered on.

Comment (vavuthu) [reply ocs_ci/ocs/node.py]: done

**Review by petr-balogh — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**